windborne/data_api.py

```diff
@@ -549,9 +549,34 @@ def get_flying_missions(output_file=None, print_results=False):
         dict: The API response containing list of flying missions.
     """
 
+    # Initial query to get total flying
+    query_params = {
+        'short': 'true',
+        'range': 'flying',
+        'include_total': 'true',
```

Comment:
This is making a request to sensor-data-api, which then makes the request to windmc without passing along these parameters, so we shouldn't include these